### DISCUSSION TITLE
Creating a node set with z=0 coordinates in Neper using -nset option

### DISCUSSION THREAD

**Discussion by PengWei97** [neperfepx/neper#695] [General]
Hi neper experts,

I am new to Neper and I am trying to create a node set called "Bottom" containing all nodes in a given mesh file (test.msh) that have a z-coordinate of zero. Because I show the following error during the calculation of moose,
```bash
*** ERROR ***
/home/*/polycrystal_2.i:265: (BCs/z_bot/boundary):
    the following node sets (ids) do not exist on the mesh: Bottom (39)
```

and the original command I created is as follows,
```
neper -T -n 4
neper -M n4-id1.tess
```

Can someone please help me with the command line to achieve this? I would greatly appreciate any suggestions or examples.

Thank you in advance for your help!

Best regards,
wei

**Comment by rquey** [comment]
The bottom node set is called `z0` in the default output.
